<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Form</class>
 <widget class="QWidget" name="Form">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>699</width>
    <height>516</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>约瑟夫环问题</string>
  </property>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>62</x>
     <y>52</y>
     <width>150</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>保存信息的文件地址：</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>62</x>
     <y>89</y>
     <width>120</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>开始报数的位置：</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_3">
   <property name="geometry">
    <rect>
     <x>62</x>
     <y>126</y>
     <width>135</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>出约瑟夫环的报数：</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="fileText">
   <property name="geometry">
    <rect>
     <x>219</x>
     <y>52</y>
     <width>171</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="startText">
   <property name="geometry">
    <rect>
     <x>219</x>
     <y>89</y>
     <width>171</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="stepText">
   <property name="geometry">
    <rect>
     <x>219</x>
     <y>126</y>
     <width>171</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="confirmButton">
   <property name="geometry">
    <rect>
     <x>520</x>
     <y>70</y>
     <width>93</width>
     <height>28</height>
    </rect>
   </property>
   <property name="text">
    <string>确认</string>
   </property>
  </widget>
  <widget class="QPushButton" name="quitButton">
   <property name="geometry">
    <rect>
     <x>520</x>
     <y>120</y>
     <width>93</width>
     <height>28</height>
    </rect>
   </property>
   <property name="text">
    <string>退出</string>
   </property>
  </widget>
  <widget class="QPlainTextEdit" name="showText">
   <property name="geometry">
    <rect>
     <x>120</x>
     <y>210</y>
     <width>441</width>
     <height>261</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label_4">
   <property name="geometry">
    <rect>
     <x>120</x>
     <y>190</y>
     <width>72</width>
     <height>15</height>
    </rect>
   </property>
   <property name="text">
    <string>输出结果</string>
   </property>
  </widget>
  <widget class="QPushButton" name="chooseButton">
   <property name="geometry">
    <rect>
     <x>410</x>
     <y>52</y>
     <width>41</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>选择</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
